src/printer.js

```diff
@@ -2472,7 +2472,15 @@ function genericPrintNoParens(path, options, print, args) {
       } else if (n.body) {
         parts.push(
           " {",
-          indent(concat([line, group(path.call(print, "body"))])),
+          indent(concat([
```

Comment:
Whoops! Need to figure out how to get format-on-save working in VSCode. It keeps looking for prettier in `node_modules` :P